BarcodeSDK.MAUI.Example/BarcodeSDK.MAUI.Example.csproj

```diff
@@ -40,6 +40,7 @@
         <MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
     </ItemGroup>
     <ItemGroup>
-      <PackageReference Include="ScanbotBarcodeSDK.MAUI" Version="7.0.2" />
+        <PackageReference Include="ScanbotBarcodeSDK.MAUI" Version="7.1.0-beta.1" />
+        <PackageReference Include="CommunityToolkit.Maui" Version="9.1.0" />

```

Comment:
Based on my recent experience. I suggest checking this example targeting `net8.0-android` and `net8.0-ios`. It may not work.